src/databricks/labs/remorph/config.py

```diff
@@ -51,13 +23,19 @@ class TranspileConfig:
     schema_name: str = "transpiler"
     mode: str = "current"
 
-    def get_read_dialect(self):
-        return get_dialect(self.source_dialect)
+    @property
+    def input_path(self):
+        if self.input_source is None:
+            raise ValueError("Missing input source!")
+        return Path(self.input_source)
+
+    @property
+    def output_path(self):
+        return None if self.output_folder is None else Path(self.output_folder)
```

Comment:
```suggestion
        return None if not self.output_folder else Path(self.output_folder)
```
This way we can also check for both empty strings and None